zqd/zeek/zeek.go

```diff
@@ -63,8 +77,8 @@ type process struct {
 	stderrBuf *bytes.Buffer
 }
 
-func newProcess(ctx context.Context, pcap io.Reader, zeekpath, outdir string) *process {
-	cmd := exec.CommandContext(ctx, zeekpath, "-C", "-r", "-", "--exec", ExecScript, "local")
+func newProcess(ctx context.Context, pcap io.Reader, args []string, outdir string) *process {
```

Comment:
Args doesn't feel like the right name for this since it includes the command name. Maybe call it command? Or change the signature to resemble os/exec.Command, like this?
```go
func newProcess(ctx context.Context, pcap io.Reader, outdir string, name string, args []string) *process {
```